docs/conf.py

```diff
@@ -145,8 +145,9 @@
 # Theme options are theme-specific and customize the look and feel of a theme
 # further.  For a list of options available for each theme, see the
 # documentation.
+html_logo="docs/aiohttp-plain.svg"
```

Comment:
Use a relative path in the logo definition
```suggestion
html_logo = "aiohttp-plain.svg"
```